src/index.js

```diff
@@ -214,12 +230,35 @@ export default class RNPickerSelect extends PureComponent {
         return {};
     }
 
+    async measureTextInput() {
```

Comment:
We don't yet allow async/await to be used in our App codebase, and I don't see it used in this component either, so can you please switch this to be a purely promise-based implementation?